src/asr_eval/utils.py

```diff
@@ -38,3 +39,80 @@ def standardize_text(text: str) -> str:
     text = remove_punctuation(text)
     text = remove_multiple_spaces(text)
     return text
+
+
+def count_words(string):
+    if isinstance(string, str):
+        return len(string.split(" "))
+    else:
+        return 1
+
+
+def count_chars(string):
+    if isinstance(string, str):
+        return len(string)
+    else:
+        return 1
+
+
+def add_error_count(
+    df: pd.DataFrame, text_col: str, condition: pd.Series | None = None
+) -> pd.DataFrame:
+    """Count words and characters and errors to calculate mean scores that are not affected by segment length"""
```

Comment:
"not affected" blir vel litt upresist, er vel mer "scores that are weighted on segment length" ?

Reply:
Hva med "Count words, characters and errors to calculate mean scores across multiple segments" ?

Reply:
```suggestion
    """Count words, characters and errors to calculate mean scores across multiple segments"""
```